client/landing/stepper/declarative-flow/new-hosted-site-flow.ts

```diff
@@ -120,7 +122,7 @@ const hosting: Flow = {
 						);
 					}
 
-					return window.location.assign( destination );
+					return navigate( 'plans' );
```

Comment:
Navigating back from checkout will get us to the processing screen without  `providedDependencies.goToCheckout` we want to go to plans in this case instead of the signup destination

Reply:
This works here because goToCheckout will always be true (apart when back from checkout) as we not present free plans! Just a reminder of this :)

Reply:
not entirely true since when create site submits when we reuse a slug we loose the id